src/swagger2har.js

```diff
@@ -247,6 +247,13 @@ var getHeadersArray = function(swagger, path, method) {
     for (var k in pathObj.parameters) {
       var param = pathObj.parameters[k]
       if (typeof param.in !== "undefined" && param.in.toLowerCase() === "header") {
+        if (typeof param["$ref"] === "string") {
+          // can't handle this yet
```

Comment:
What functionality are we skipping / do we not support here?

Reply:
swagger2har does not currently support refs to external files.  This check
for `/^http/` is seen in other places in code before doing resolveRef. 

Maybe it would be more clear if I moved the comment down a line

Reply:
non-blocking, thanks for the explanation. Good info to put in the CHANGELOG :-)